src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -141,6 +148,11 @@ export default {
 	methods: {
 		hideMenu() {
 			this.openedMenu = false
+			this.$emit('update:menu-open', this.openedMenu)
```

Comment:
Does this work really?
I knew vue was auto-translating kebab-case to camelcase when using props, but not on events! :thinking:

Reply:
The [documentation](https://vuejs.org/v2/guide/components-custom-events.html#Event-Names) says:
> [...] For these reasons, we recommend you **always use kebab-case for event names**.

However, it looks that in this special case (using a prop with sync modifier), the update event can use both kebab-case or camelCase. And you can use both kebab-case or camelCase as HTML attribute when using the component. At least my test cases are working.

Therefore, I'm fine with changing the syntax. Please have a look at the added commit. This works fine for me in the notes app.

Reply:
if the Vue docs recommend kebab-case then we should use it

Reply:
Agreed then! SOrry abou tthat :)
Thanks for the tip @korelstar !